### DISCUSSION TITLE
About sucking out save data of Famicom

### DISCUSSION THREAD

**Discussion by hikami99** [sanni/cartreader#357] [Need help]
I have a question.
Please tell me how to extract the save data of Famicom with the cartridge reader (Save the Hero Project).

**Comment by SAVE-THE-HERO** [comment]
Hello.
The product pictured in your icon is only available in Japan.
I'm a bit mystified.
See links below.
If you have any further questions, please contact me at the email address given on the sales page.
This is good If you are Japanese
https://www.youtube.com/watch?v=KpU3NPhORw0
Check the settings and try "Read RAM".
This will not work if the cartridge has a dead battery or no data.
Thank you

**Reply by hikami99** [reply]
I appreciate your prompt reply.
I'll give it a try.
